### PR TITLE
Bug fix: #14014 - p-tooltip : autohide="false" does not work every time

### LINKED ISSUE #14014: p-tooltip : autohide="false" does not work every times
### Describe the bug

**add [autohide]="false" to the p-tooltip component and hover it from the arrow carret element**

### It doen't works when mouse hover the tooltip by the carret arrow : 
ko

### It works when mouse hover the tooltip directly (by the bottom or top of the tooltip) : 
ok

### Environment

primeng 16.7.0

### Reproducer

_No response_

### Angular version

16

### PrimeNG version

16.7.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18

### Browser(s)

_No response_

### Steps to reproduce the behavior

_No response_

### Expected behavior

_No response_

### PR DESCRIPTION
**Pull request by maruthumj**
fixes #14014

### CODE REVIEW

**Review by SoyDiego — APPROVED** [review]

File: src/app/components/tooltip/tooltip.ts [hunk 1]
```diff
@@ -297,7 +297,13 @@ export class Tooltip implements AfterViewInit, OnDestroy {
 
     onMouseLeave(e: MouseEvent) {
         if (!this.isAutoHide()) {
-            const valid = DomHandler.hasClass(e.target, 'p-tooltip') || DomHandler.hasClass(e.target, 'p-tooltip-arrow') || DomHandler.hasClass(e.target, 'p-tooltip-text') || DomHandler.hasClass(e.relatedTarget, 'p-tooltip');
+            const valid =
+                DomHandler.hasClass(e.target, 'p-tooltip') ||
```
Comment (SoyDiego) [inline src/app/components/tooltip/tooltip.ts:301]: Your code works perfectly thanks!
If you leave without `e.target` works good too and less lines :)

image

Comment (maruthumj) [reply src/app/components/tooltip/tooltip.ts]: thanks for the suggestion @SoyDiego. I have pushed the changes!

Comment (SoyDiego) [reply src/app/components/tooltip/tooltip.ts]: Happy to help 🚀

**Review by cetincakiroglu — APPROVED** [review]
Thanks for the effort and support!

**Pull request merged**